src/controller/clients/condo-search-client.js

```diff
@@ -86,6 +87,8 @@ class CondoSearchClient extends BaseClient {
       if(this.store.state.parcels.pwd === null) {
         const latLng = {lat: feature.geometry.coordinates[1], lng: feature.geometry.coordinates[0]}
         const callback = () => {
+
+          feature.properties.opa_owners = ["Condominium (" + totalUnits + " Units)"];
           feature.properties.street_address = this.store.state.parcels.pwd.properties.ADDRESS;
           feature.properties.opa_address = this.store.state.parcels.pwd.properties.ADDRESS;
           feature.properties.pwd_parcel_id = this.store.state.parcels.pwd.properties.PARCELID;
```

Comment:
It looks like these four lines, starting at `feature.properties.opa_owners = ...` and the four lines in the else condition below can be extracted out of the if / else, possible into a separate method - something like `setFeatureProperties()`...